docs/mllib-linear-methods.md

```diff
@@ -78,6 +78,10 @@ methods `spark.mllib` supports:
   </tbody>
 </table>
 
+A binary label y is denoted as either +1 (positive) or −1 (negative), which is
```

Comment:
Yeah back to the start, sorry.  Looks good, though you can also remove the statement below that this duplicates now. Thank you.